frontend/routes/index.tsx

```diff
@@ -9,9 +9,18 @@ import { Head } from "$fresh/runtime.ts";
 import { ComponentChildren } from "preact";
 import { HomepageHero } from "../components/HomepageHero.tsx";
 import { Logo } from "../components/Logo.tsx";
+import { NewsCard } from "../components/NewsCard.tsx";
+
+interface Post {
+  title: string;
+  description: string;
+  image: string;
+  url: string;
```

Comment:
This seems to be undefined. I can't actually click on the cards :sweat_smile:

Reply:
Whoops, we call it `link` on dotcom, not `url`. Fixed now. 👍